.github/workflows/npm-publish.yml

```diff
@@ -0,0 +1,23 @@
+name: Node.js Package
+
+on: [ release ]
+
+jobs:
+
+  publish-npm:
+
+    runs-on: ubuntu-latest
+
+    needs: test
+
+    steps:
+      - uses: actions/checkout@v1
+      - uses: actions/setup-node@v1
+        with:
+          node-version: 12
+          registry-url: https://registry.npmjs.org/
+      - run: npm ci
+      # ensures latest dist/ through build in prepublishOnly
+      - run: npm publish
+        env:
+          NODE_AUTH_TOKEN: ${{secrets.npm_token}}
```

Comment:
```suggestion
          NODE_AUTH_TOKEN: npm_token
```

Reply:
added token and named in a very surprising way like suggested above

Reply:
unless I am suggesting crap and actually the code above is fine and will be processed as intended by github

Reply:
Is effectively a [copy from here](https://github.com/actions/starter-workflows/blob/master/ci/npm-publish.yml#L33) but I would not mind you being right.

Reply:
ignore the suggestion please